stable/traefik/values.yaml

```diff
@@ -103,6 +103,8 @@ ssl:
   # cipherSuites: []
   # https://docs.traefik.io/configuration/entrypoints/#specify-minimum-tls-version
   # tlsMinVersion: VersionTLS12
+  # https://docs.traefik.io/configuration/entrypoints/#strict-sni-checking
+  sniStrict: false
```

Comment:
Can you make this configuration optional? (commented out)?

Reply:
I thought the values.yml is just a default value?  So I wasn't sure if it was best to explicitly set it or not.  Either way I commented this out to match the others.